README.rst

```diff
@@ -254,6 +254,66 @@ To configure any Fedora 20 bare metal machine (e.g. on OnMetal as above)::
 
 Where ``${PASSWORD}`` is the password in ``slaves.fedora-vagrant.passwords`` from the ``config.yml`` or ``staging.yml`` file used to deploy the BuildBot master on hostname or IP address ``${MASTER}``.
 
+PistonCloud
+-----------
+
+The following builders need to run on Centos-7 on PistonCloud:
+
+* flocker-openstack-pistoncloud
+
+To create this machine you'll need to access various machines within pistoncloud via an "SSH jump host".
+
+The machines are referred to here as:
```

Comment:
I don't actually know if they're physical machines. But I will make it clearer that the third machine does not yet exist.